src/main/scala/is/hail/expr/ir/functions/ArrayFunctions.scala

```diff
@@ -189,13 +196,11 @@ object ArrayFunctions extends RegistryFunctions {
       val body =
         Let(value, ArrayRef(a, Ref(idx, TInt32())),
           Let(m, GetField(Ref(accum, tAccum), "m"),
-            If(IsNA(Ref(value, t)),
-              Ref(accum, tAccum),
-              If(IsNA(Ref(m, t)),
+            If(IsNA(Ref(m, t)),
```

Comment:
I haven't changed argmax/argmin at all, I deleted the filter_missing flags. They would lead to weird behavior -- they'd return the index of the min in the array after missingness was removed, which is totally not what people would want. So gone for now!

Reply:
I think you still have to put back in the `If(IsNA(Ref(value, t)),` case. (taking `argmin(0, na, 1) = 0` as an example, what you would have would currently return `2` since the `na` would cause the accumulator to get set as missing).

Reply:
(I'm pretty sure this is the case, but it would also be a good test case, I think)